nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Cli/Program.cs

```diff
@@ -20,6 +21,9 @@ internal static async Task<int> Main(string[] args)
         };
         command.TreatUnmatchedTokensAsErrors = true;
 
+        // trim quotes
+        args = args.Select(x => x.Trim('"')).ToArray();
```

Comment:
Are there double quotes coming through here?  I thought the dotnet argument parsing code and/or OS would trim them off before they get here.